src/views/home.ejs

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Weather</title>
```

Comment:
Im assuming this is being borrowed from your School project? Title would need to be changed to OrganizeMe